app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hiden: bool = False) -> None:
```

Comment:
There's a typo in the parameter name `hiden`. It should be `hidden` to match the attribute name used in the class.